js/invoice.js

```diff
@@ -0,0 +1,24 @@
+export const InvoiceAPI = superclass =>
+    class extends superclass {
+        async createInvoice(payload) {

```

Comment:
Add `options` parameter that defaults to `{}` and pass it along to the request (similar to UPS API JS)